docs/tutorials/fundamentals/viewer.md

```diff
@@ -97,60 +79,76 @@ nbscreenshot(viewer, alt_text="photograph of an astronaut in napari viewer")
 
 ## Layout of the viewer
 
-The viewer is organized into a few key areas:
+The viewer is organized into a few key areas which are explained in the next sections:
+* Main Menu (top bar menu)
+* Layer Controls
+* Layer Buttons
+* Layer List
+* Viewer Buttons
+* Status Bar
+* Canvas
+* Dimension Sliders
+* Scroll Buttons
+* Frame Playback  
+
+The image below has the areas of the viewer labeled: 
 
-![image: viewer layout](../assets/tutorials/viewer_layout.jpg)
+  ![image: viewer layout](../assets/tutorials/Viewer-with-arrows.png)
 
 We'll go through each of these in the next sections.
 
-+++
+### Main menu or top bar menu
 
-### Canvas
+The main menu consists of the **File**, **View**, **Window**, **Plugins**, and **Help** options which are illustrated in this section.  
+
+* **File** contains the options shown in the graphic below: 
+![image: File Menu](../assets/tutorials/main-menu-file.png)  
 
-The canvas is in the center of the viewer and contains the visual display
-of the data passed to **napari**, including images, point, shapes, and our other
-supported data types. Under the hood the canvas is a `vispy.scene.SceneCanvas`
-object which has built-in support for features such as zooming and panning. As
-`vispy` uses `OpenGL` and your graphics card, panning and zooming are highly
-performant. You can also return to the original zoom level by clicking the
-`home` button in the viewer buttons panel.
+    All the options on the **File** menu are self-explanatory except **Preferences**. **Preferences** has the options shown below. To learn more about the **Preferences** menu, visit this [tutorial](https://napari.org/stable/guides/preferences.html).  
+    ![image: Preferences sub-menu](../assets/tutorials/main-menu-file-sub-menu-preferences.png)  
+    </br>
+    
 
-+++
+* **View** contains the options shown in the graphic below:  
+![image: View Menu](../assets/tutorials/main-menu-view.png)    
+    </br>
+  
 
-![image: pan and zoom with napari](../assets/tutorials/viewer_pan_zoom.webm)
+* **Window** contains the options shown in the graphic below:   
+![image: Window Menu](../assets/tutorials/main-menu-window.png)    
+    </br>
 
-+++
+* **Plugins** contains the options shown in the graphic below:   
+![image: Plugins Menu](../assets/tutorials/main-menu-plugins.png)    
+    </br>
+
+* **Help** contains the options shown in the graphic below:   
+![image: Help Menu](../assets/tutorials/main-menu-help.png)    
+    </br>
+
+### Canvas
+
+The **canvas** is in the center of the viewer and contains the visual display of the data passed to **napari**, including images, point, shapes, and other supported data types. Under the hood the canvas is a `vispy.scene.SceneCanvas` object which has built-in support for features such as zooming and panning. As `vispy` uses `OpenGL` and your graphics card, panning and zooming are highly performant. You can return to the original zoom level by clicking the `home` button in the viewer buttons panel.
```

Comment:
images, points, and shapes are not capitalized because they are not capitalized in napari and I thought it best to be consistent with the interface. I added the back ticks and the comma, as you suggested.  (You are doing a great job of going over these docs.  This is great feedback.)

Reply:
I guess the question is are we using the terms generically or specific to napari layers.
In other docs, like the getting started guide, when referring to layers, they are capitalized. Even in this one, if you scroll down to Layer controls.